src/components/OSCALControlModification.js

```diff
@@ -19,11 +19,15 @@ const OSCALControlModificationsButton = styled(IconButton)(
  * Create a typography html object for addsOrRemovesLabel
  *
  * @param {Object} addsElements add or remove object to map into html
- * @param {String} addsLabel boolean variable, true if adding
+ * @param {String} addsRemovesLabel string variable
```

Comment:
I think saying the string variable `addsRemovesLabel` is a string variable is not super descriptive. Please update this.

I also am not sure I love that variable name either, but am blanking on a better one right now.